snap/snapcraft.yaml

```diff
@@ -155,27 +151,27 @@ parts:
       cp -v load-snap-options.sh $CRAFT_PART_INSTALL/bin/
 
 apps:
-  lighting:
-    daemon: simple
-    command-chain:
-      - bin/load-snap-options.sh
-    command: bin/run.sh
-    install-mode: disable
-    plugs:
-      - network
-      - network-bind
-      - bluez
-      - avahi-control
-      - gpio-memory-control
-      - otbr-dbus-wpan0
-
-  help:
-    command: bin/lighting-app -h
+  # lighting:
+  #   daemon: simple
+  #   command-chain:
+  #     - bin/load-snap-options.sh
+  #   command: bin/run.sh
+  #   install-mode: disable
+  #   plugs:
+  #     - network
+  #     - network-bind
+  #     - bluez
+  #     - avahi-control
+  #     - gpio-memory-control
+  #     - otbr-dbus-wpan0
+
+  # help:
+  #   command: bin/lighting-app -h
 
   # This app is to test the GPIO control without using a Matter controller
   test-blink:
     command-chain:
       - bin/load-snap-options.sh
     command: bin/test-blink
     plugs:
-      - gpio-memory-control
+    - dev-gpiochip0
```

Comment:
This doesn't match the defined plug name.

Reply:
Fixed!